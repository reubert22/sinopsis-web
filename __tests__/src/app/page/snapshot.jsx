import { render } from '@testing-library/react'
import Home from '@/app/page'

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  var a

  expect(container).toMatchSnapshot()
})
