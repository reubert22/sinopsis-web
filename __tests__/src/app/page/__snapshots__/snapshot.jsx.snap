// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24"
  >
    <div
      class="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex"
    >
      <div
        class="fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none"
      >
        <a
          class="pointer-events-none flex place-items-center gap-2 p-8 lg:pointer-events-auto lg:p-0"
          href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          By
           
          <img
            alt="Vercel Logo"
            class="dark:invert"
            data-nimg="1"
            decoding="async"
            fetchpriority="high"
            height="24"
            src="/vercel.svg"
            style="color: transparent;"
            width="100"
          />
          and TMDB
        </a>
      </div>
    </div>
    <div
      class="relative flex place-items-center before:absolute before:h-[300px] before:w-full sm:before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-full sm:after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700 before:dark:opacity-10 after:dark:from-sky-900 after:dark:via-[#0141ff] after:dark:opacity-40 before:lg:h-[360px] z-[-1]"
    >
      <div
        class="flex items-end"
      >
        <p
          class="relative dark:drop-shadow-[0_0_0.3rem_#000] text-white text-5xl"
        >
          SINOPSIS
        </p>
        <p
          class="relative dark:drop-shadow-[0_0_0.3rem_#000] text-white text-xl"
        >
          WEB
        </p>
      </div>
    </div>
    <div
      class="mb-32 grid text-center lg:max-w-5xl lg:w-full lg:mb-0 lg:grid-cols-4 lg:text-left"
    >
      <a
        class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
        href="/trending"
      >
        <h2
          class="mb-3 text-2xl font-semibold"
        >
          Trending
           
          <span
            class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none"
          >
            -&gt;
          </span>
        </h2>
        <p
          class="m-0 max-w-[30ch] text-sm opacity-50"
        >
          Find in-depth information about Next.js features and API.
        </p>
      </a>
      <a
        class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
        href="trending"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="mb-3 text-2xl font-semibold"
        >
          Movies
           
          <span
            class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none"
          >
            -&gt;
          </span>
        </h2>
        <p
          class="m-0 max-w-[30ch] text-sm opacity-50"
        >
          Learn about Next.js in an interactive course withÂ quizzes!
        </p>
      </a>
      <a
        class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
        href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="mb-3 text-2xl font-semibold"
        >
          Tv Series
           
          <span
            class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none"
          >
            -&gt;
          </span>
        </h2>
        <p
          class="m-0 max-w-[30ch] text-sm opacity-50"
        >
          Explore starter templates for Next.js.
        </p>
      </a>
      <a
        class="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
        href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <h2
          class="mb-3 text-2xl font-semibold"
        >
          All
           
          <span
            class="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none"
          >
            -&gt;
          </span>
        </h2>
        <p
          class="m-0 max-w-[30ch] text-sm opacity-50 text-balance"
        >
          Instantly deploy your Next.js site to a shareable URL with Vercel.
        </p>
      </a>
    </div>
  </main>
</div>
`;
